firepit/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "firepit",
```

Comment:
Will the name "firepit" be user-visible anywhere?

Reply:
Nah, wont be visible, this package.json can be viewed as just a holder for the npm scripts, it's never published and never seen unless you're looking for it.